mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BottomSheetAssetSortList.tsx

```diff
@@ -56,45 +55,30 @@ export const BottomSheetAssetSortList = ({
     index: number
   }): JSX.Element => {
     return (
-      <ThemedTouchableOpacity
-        style={tailwind('px-4 py-3 flex-row justify-between')}
+      <ThemedTouchableOpacityV2
+        dark={tailwind('border-mono-dark-v2-300')}
+        light={tailwind('border-mono-light-v2-300')}
+        style={tailwind('px-5 py-3 flex-row  items-center justify-between border-b-0.5 py-2.5', { 'border-t-0.5': index === 0 })}
         testID={`select_asset_${getDisplayedSortText(item)}`}
         key={index}
         onPress={() => {
           onButtonPress(item)
         }}
       >
-        <ThemedText>
+        <ThemedTextV2
+          style={tailwind('py-2 text-sm font-normal-v2')}
+        >
           {translate('screens/PortfolioScreen', getDisplayedSortText(item))}
-        </ThemedText>
+        </ThemedTextV2>
         {selectedAssetSortType === item && (
           <ThemedIcon
-            size={24}
-            name='check'
+            size={18}
```

Comment:
to compensate icon's "internal padding"
```suggestion
            size={20}
```